### DISCUSSION TITLE
Flow chart and hierarchy components

### DISCUSSION THREAD

**Discussion by conradgags** [Budibase/budibase#15176] [Help]
Hi,

Firstly, I'm loving Budibase. I realise I've been looking for this platform for many years.

I'm looking for a component to be able to do flowcharts and hierarchy charts in budibase. I got to know React Flow a few years ago and I notice that the Automation configurator and the Role Manager look like they use similar functionality.

Does anyone have ideas on a component to provide the above?

**Comment by mikesealey** [comment]
Hey @conradgags 

Looking at the [Curated List of Custom Plugins](https://github.com/Budibase/plugins) there doesn't seem to be anything that matches what you're looking for. You may have more luck asking on our [Discord server](https://discord.com/invite/ZepTmGbtfF) in the Contribue >> Components channel. 

If you're still drawing a blank there, you can always submit a new feature request [here](https://github.com/Budibase/budibase/issues/new/choose). We get together once per week to review all new issues submitted, including feature requests. If you took some time to work on what your feature request might look like, how it would behave, and how it would interact with the rest of Budibase, we would certainly consider it.
